gateway/1.9/modules/ROOT/pages/policies-outbound-basic-auth.adoc

```diff
@@ -0,0 +1,89 @@
+= Credential Injection Basic Authentication Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: basic authentication, api gateway, flex gateway, gateway, policy
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | Credential Injection Basic Authentication
+>s|Summary      | Injects a basic authorization header into outgoing requests
+>s|Category | Security
+>s|First Flex Gateway version available | v1.9.0
+>s| Returned Status Codes | No return codes exist for this policy. Error codes are returned from the upstream service.
+|===
+
+== Summary
+
+The Credential Injection Basic Authentication policy injects an `Authorization` header containing a username and password into outbound requests. This policy ensures that requests sent from Flex Gateway to an upstream service include the correct authentication headers. The Authorization header is formatted as `Authorization: Basic <username:password>`.
+
+The policy follows https://tools.ietf.org/html/rfc7617[Basic HTTP authentication standards^].
+
+When an error is encountered, the policy returns an "WWW-Authenticate" HTTP header field.
+
+== Configuring Policy Parameters
+
+=== Flex Gateway Local Mode
+
+In Flex Gateway Local Mode, you apply the Basic Authentication: Simple policy to your API via declarative configuration files. Refer to the following policy definition and table of parameters:
+
+----
+- policyRef:
+    name: http-basic-authentication-flex
+  config:
+    username: <string> // OPTIONAL
+    password: <string> // OPTIONAL
+    overwrite: <boolean> // OPTIONAL
+    customHeader: <string> // OPTIONAL
+----
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+|===
+|Parameter | Required or Optional | Default Value | Description
+
+| `username`
+| Optional
+| N/A
+| The username for the basic authentication credentials.
+
+| `password`
+| Optional
+| N/A
+| The password for the basic authentication credentials.
+
+| `overwrite`
+| Optional
+| true
+| When enabled, if the `Authorization` header is present, the policy overwrites it with the configured credentials.
+
+| `customHeader`
+| Optional
+| N/A
+| When configured, a custom header is injected instead of the `Authorization` header. If *Overwrite header* is enabled, the policy overwrites the custom header is present instead of the `Authorization` header.
```

Comment:
```suggestion
| When configured, a custom header is injected instead of the `Authorization` header. If *Overwrite header* is enabled, the policy overwrites the custom header if present instead of the `Authorization` header.
```

Reply:
Not sure if this is the correct suggestion.